pkg/tests/stamp_test.py

```diff
@@ -13,17 +13,52 @@
 # limitations under the License.
 """Test time stamping in pkg_tar"""
```

Comment:
And now `pkg_zip`.  Perhaps make this docstring generic?

Reply:
Done